apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/RDATestUtils.java

```diff
@@ -0,0 +1,222 @@
+      ImmutableList.of(
+          PreAdjFissProcCode.class,
+          PreAdjFissClaim.class,
+          PreAdjMcsDetail.class,
+          PreAdjMcsDiagnosisCode.class,
+          PreAdjMcsClaim.class);
+
+  public static final String PERSISTENCE_UNIT_NAME = "gov.cms.bfd.rda";
+
+  private EntityManager entityManager;
+
+  public void init() {
+    final DataSource dataSource = DatabaseTestUtils.get().getUnpooledDataSource();
+    DatabaseSchemaManager.createOrUpdateSchema(dataSource);
+
+    final Map<String, Object> hibernateProperties =
+        ImmutableMap.of(org.hibernate.cfg.AvailableSettings.DATASOURCE, dataSource);
+
+    entityManager =
+        Persistence.createEntityManagerFactory(PERSISTENCE_UNIT_NAME, hibernateProperties)
+            .createEntityManager();
+  }
+
+  public void seedData(Collection<?> entities) {
+    doTransaction(em -> entities.forEach(em::persist));
+  }
+
+  public void truncateTables() {
+    doTransaction(
+        em -> {
+          TABLE_ENTITIES.forEach(
+              e -> em.createQuery("delete from " + e.getSimpleName() + " f").executeUpdate());
+        });
+  }
+
+  public void doTransaction(Consumer<EntityManager> transaction) {
+    entityManager.getTransaction().begin();
+    transaction.accept(entityManager);
+    entityManager.getTransaction().commit();
+  }
+
+  public String expectedResponseFor(String requestId) {
+    StringBuilder expectedResponse = new StringBuilder();
+    InputStream fileStream =
+        this.getClass()
+            .getClassLoader()
+            .getResourceAsStream("endpoint-responses/v2/" + requestId + ".json");
+
+    if (fileStream != null) {
+      try (BufferedReader br = new BufferedReader(new InputStreamReader(fileStream))) {
+
+        String line;
+        while ((line = br.readLine()) != null) {
+          expectedResponse.append(line.trim());
+        }
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+    }
+
+    return expectedResponse.toString();
+  }
+
+  public List<PreAdjFissClaim> fissTestData() {
+    return Arrays.asList(fissTestDataA(), fissTestDataB());
+  }
+
+  private PreAdjFissClaim fissTestDataA() {
+    PreAdjFissClaim claim = new PreAdjFissClaim();
+    claim.setDcn("123456");
+    claim.setHicNo("hicnumber");
+    claim.setCurrStatus('a');
+    claim.setCurrLoc1('z');
+    claim.setCurrLoc2("Somda");
+    claim.setMedaProvId("meda12345");
+    claim.setTotalChargeAmount(new BigDecimal("1234.32"));
+    claim.setReceivedDate(LocalDate.ofEpochDay(0));
+    claim.setCurrTranDate(LocalDate.ofEpochDay(1));
+    claim.setAdmitDiagCode("admitcd");
+    claim.setPrincipleDiag("princcd");
+    claim.setNpiNumber("8876543211");
+    claim.setMbi("123456MBI");
+    claim.setMbiHash("a7f8e93f09");
+    claim.setFedTaxNumber("abc123");
+    claim.setLastUpdated(Instant.ofEpochMilli(0));
+    claim.setStmtCovFromDate(LocalDate.ofEpochDay(200));
+    claim.setStmtCovToDate(LocalDate.ofEpochDay(200));
```

Comment:
Any reason not to use the lombok builders for all of these test data objects in this class?